### PR TITLE
search bar completed. removed search bar from head

### PR DESCRIPTION
**Pull request by TomBrockington**
have completed and added the search bar for students to the tacher admin panel.
removed the search bar from header
added the endpoint /user/:id/profile to app.js

### CODE REVIEW

File: src/components/teacher/components/SearchBar.jsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  let navigate = useNavigate();
+
+  // handle the text input to search bar
+  const handleChange = event => {
+    event.preventDefault();
+    const { value } = event.target;
+    setInputText({
+      value,
+    });
+    console.log('search input', inputText);
```
Comment (Auenc) [inline src/components/teacher/components/SearchBar.jsx:22]: Let's remove this console log

File: src/components/teacher/components/SearchBar.jsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+        const users = res.data.data.users;
+        console.log('users:', users);
+        console.log('input:', inputText);
+
+        const foundUser = users.filter(user =>
+          user.first_name.toLowerCase().includes(inputText.value.toLowerCase())
+        );
+
+        console.log('found', foundUser);
```
Comment (Auenc) [inline src/components/teacher/components/SearchBar.jsx:39]: Let's remove this console log

File: src/components/teacher/components/SearchBar.jsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+        setSearchResult(foundUser);
+      })
+
+      .catch(err => console.log(err.response));
+  };
+
+  const routeChange = id => {
+    localStorage.setItem('id', id);
+    console.log('routing', id);
```
Comment (Auenc) [inline src/components/teacher/components/SearchBar.jsx:49]: Same as above

**Review by Auenc — CHANGES_REQUESTED** [review]
Just some console.logs to remove

**Review by Auenc — APPROVED** [review]

**Review by richard-mattinson — APPROVED** [review]

**Pull request merged**